packages/api/src/routes/user.ts

```diff
@@ -302,4 +311,45 @@ router.get("/connect", async (req: Request, res: Response) => {
   return res.status(status.OK).json(users);
 });
 
+/**
+ * Removes the device from the user's profile.
+ */
+async function removeDevice(req: Request, userId: string) {
+  const cxId = getCxIdOrFail(req);
+  const connectedUser = await getConnectedUserOrFail({ id: userId, cxId });
+  const provider = getFromQuery("provider", req);
+  const deviceId = getFromQuery("deviceId", req);
+
+  if (!provider || !deviceId) throw new BadRequestError();
```

Comment:
Let's add a description to the Error, something like "either provider or deviceId must be provided"